webui/react/src/pages/WorkspaceDetails/WorkspaceMembers.module.scss

```diff
@@ -1,25 +0,0 @@
-.base {
-  height: fit-content;
-}
-.userRowItem {
-  display: inline-block;
-  vertical-align: middle;
-
-  img {
-    margin-right: 10px;
-  }
-  i {
-    margin-right: 10px;
-  }
-}
-.userAvatarRowItem {
-  align-items: center;
-  display: inline-flex;
-  height: 100%;
-  line-height: 100%;
-  margin-right: 10px;
-  vertical-align: middle;
-}
-.membersContainer {
-  margin-top: 30px;
-}
```

Comment:
This file seems to be a partial copy of `webui/react/src/pages/WorkspaceDetails/WorkspaceProjects.module.scss`, and is not used